doc/.readthedocs.yaml

```diff
@@ -0,0 +1,27 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+formats:
+  - pdf
+  - epub
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+# Build user documentation in the doc/ directory with Sphinx
+sphinx:
+  configuration: doc/rst/conf.py
+  #fail_on_warning: true
+
+# We recommend specifying your dependencies to enable reproducible builds:
+# https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
+python:
+  install:
+  - requirements: doc/requirements.txt
```

Comment:
For future reference, the `docutils` version constraint we have in the requirements file _might_ not be necessary anymore (this build in my test repo without it). I left it here as it still works, and we may want to use this file for reproducible builds in the future.